rc/base/spell.kak

```diff
@@ -9,6 +9,10 @@ def -params ..1 spell -docstring "Check spelling of the current buffer with aspe
         printf %s\\n "set buffer spell_tmp_file ${file}"
     }
     %sh{
+        _echo() {
```

Comment:
Calls to `printf` tend to clutter the lines unecessarily, so whenever there's more than a couple calls to it, I create an `_echo` function. Users know what the function does, it's 7 byte less long than its `printf` counterpart, and it simplifies the code.

Why don't you want this function in ? Not enough calls to `printf` to replace with it ?